default_hostgroup.yaml.example

```diff
@@ -18,6 +18,8 @@
 ---
 :default_hostgroup:
   :facts_map:
+    "Go_agents":
+      "hostname": ["go_agent1", "go_agent2", "go_agent[1][1-3]]"
```

Comment:
This is not the ideal way to specify an array in YAML (although it does work). Better to use

```
"Go_agents":
- "go_agent1"
- "go_agent2"
- "go_agent[1][1-3]"
```